pyqg/model.py

```diff
@@ -315,7 +319,7 @@ def stability_analysis(self,bottom_friction=False):
 
         Q =  I[:,:,np.newaxis,np.newaxis]*(self.ikQy - self.ilQx).imag
 
-        Uk =(self.Ubg*I)[:,:,np.newaxis,np.newaxis]*self.k
+        Uk =(self.Ubg*I[:,:,np.newaxis])[:,:,:,np.newaxis]*self.k
```

Comment:
I think we need the docstring to be modified somehow to reflect the changes to `Ubg`. Has the API changed? if not, how does one specify the background flow?

Reply:
@rabernat I can absolutely update the doctoring and API docs. The API here has been extended. So, we can specify a Ubg via U, or U1/U2 parameters passed into the model that are rank 1 arrays, allowing for meridional structure. I overloaded this parameter in such a way that you can also continue using a scalar/constant background flow, making the updates backwards compatible.